attribute-masking-service/README.md

```diff
@@ -16,13 +16,18 @@ limitations under the License.
 
 # <img src="../logos/logo.svg" width="180">
 
-# Palisade Attribute-Masking-Service
+# Flow of Control
 
-The attribute-masking-service is the final transformation the palisade system applies to resources before they are returned.
-The service performs two functions:
-* Store the full details of the authorised request in a persistence store, to be later retrieved by the data-service
-* Mask the leafResource, removing any sensitive information
+![Attribute-Masking Service Diagram](doc/attribute-masking-service.png)
 
+The Attribute-Masking Service will persist authorised requests for later use by the Data Service and modify the request that is forwarded to the Topic-Offset Service and Filtered-Resource Service.
```

Comment:
I would recommend saying that it's forwarded to the Kafka topic rather than the service.

Reply:
I did that on purpose and would like to keep it this way.  This section is meant to be a very light overview in "English".  It is meant to give an understanding of what it does in a few sentences. It should give you an understanding of the service without having to know about the different model types and Kafka.  Later, I essentially talk about the same thing but with the technical detail included.